src/string.js

```diff
@@ -153,5 +156,5 @@ String.prototype.isDigit = function () {
  * @returns {String}
  */
 String.prototype.doubleCheck = function () {
-    return /(\S)\1/g.test(this);
+    return /(.)\1/.test(this);
```

Comment:
### Nitpick

The task was to match an occurrence of "any" double repeated character. "." does not always match new lines, do some research on [\s\S]